### DISCUSSION TITLE
You developers don't even bother to follow your own rules.

### DISCUSSION THREAD

**Discussion by luciph0r** [LSPosed/LSPosed#2000] [General]
Why was [my bug report](https://github.com/LSPosed/LSPosed/issues/1998) closed when I proved every requested bit of information **including the damn Magisk logs**? You can't be bothered to read an issue properly and just dismiss it?

> For usage issues, please provide the log zip saved from manager. Without log, the issue will be closed.

> You don't read the report guide carefully so this issue is invalid and closed silently. Lock because of off-topic.

No, geniuses @vvb2060  @yujincheng08  - **You** both didn't bother to see that I _have_ attached the log to the issue, or chose to ignore it. I wouldn't even have been able to submit the report if I hadn't included it. Or do you think one can submit a form without entering a mandatory field?

If you don't give a crap about the fact that your module is unusable where Safetynet attestation is required then say so instead of outright lying like this.

**Comment by yujincheng08** [comment]
Poor kid dont even understand english.

log zip *saved from manager*.

**Comment by yujincheng08** [comment]
![IMG_20220621_122014_725](https://user-images.githubusercontent.com/5022927/174715359-7c713027-e2d4-470a-abab-21be2f0c7b80.jpg)
and, FYI

**Comment by vvb2060** [comment]
We can't even understand why you are uploading magisk logs...

**Comment by yujincheng08** [comment]
maybe this kid thinks we are magisk devs so choose to upload magisk log instead of LSPosed log. lol

**Comment by canyie** [comment]
"Manager" = "LSPosed Manager".
BTW, there is no something called "Magisk Manager". It's "Magisk app".

**Comment by vvb2060** [comment]
https://github.com/LSPosed/LSPosed/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml  
Search "issue will be closed", there are two places in total, you have not done any of them.
